.env.schema

```diff
@@ -1,4 +1,4 @@
-app__can_main_net_url=http://18.163.57.224:8888
+app__can_main_net_url=http://api-testnet.canfoundation.io:8888
```

Comment:
actually, we don't need to specify the value in the schema file https://www.npmjs.com/package/dotenv-extended

Reply:
i already reverted it! @manh-vv ma
help me check again!